### PR TITLE
JS bug on 404 page fix

### PR DESCRIPTION
**Pull request by tbrandles**

### CODE REVIEW

File: crate/pages/404.html [hunk 1]
```diff
@@ -24,9 +24,7 @@
 </div>
 
 <script type="text/javascript">
- (function($){
-   ga('send', 'event', 'error', '404', location.pathname, 1);
- }(jQuery));
+  ga('send', 'event', 'error', '404', location.pathname, 1);
```
Comment (chaudum) [inline crate/pages/404.html]: `ga` is not defined at that point in time - throws error

Comment (chaudum) [inline crate/pages/404.html]: best would be to put it into the extra_js block::

```
{% block extra_js %}
<script type="text/javascript">
  $(function(){
    ga('send', 'event', 'error', '404', location.pathname, 1);
  });
</script>
{% endblock %}
```

**Comment by tbrandles** [comment]
@chaudum could you take a look? Thanks

**Comment by chaudum** [comment]
ack

**Pull request merged**